step.yml

```diff
@@ -58,39 +55,55 @@ inputs:
 - command: install
   opts:
     title: CocoaPods command
-    description: |-
-      Use pod `install` to download the explicit version listed in the Podfile.lock without trying to check if a newer version is available.
-      Use pod `update` to update every Pod listed in your Podfile to the latest version possible.
+    summary: CocoaPods command to use for installing dependencies.
+    description: |
+      CocoaPods command to use for installing dependencies.
+
+      Available options:
+      - `install`: Use `pod install` to download the explicit version listed in the Podfile.lock without trying to check if a newer version is available.
+      - `update`: Use `pod update` to update every Pod listed in your Podfile to the latest version possible.
     is_required: true
     value_options:
     - install
     - update
 - source_root_path: $BITRISE_SOURCE_DIR
   opts:
     title: Workdir
-    description: |-
+    summary: Directory path where the project's Podfile (and optionally Gemfile) is placed.
+    description: |
       Directory path where the project's Podfile (and optionally Gemfile) is placed.
+
       CocoaPods commands will be executed in this directory.
     is_required: true
 - podfile_path: ""
   opts:
     title: Podfile path
+    summary: Path of the project's Podfile.
     description: |-
+      Path of the project's Podfile.
+
       By specifying this input `Workdir` gets overriden by the provided file's directory path.
 - verbose: "false"
   opts:
-    title: Execute all CocoaPods commands in verbose mode?
+    title: Enable verbose logging
+    summary: Execute all CocoaPods commands in verbose mode?
     description: |
+      Execute all CocoaPods commands in verbose mode?
```

Comment:
Having a question in the summary and description looks a bit weird to me. I would make it a statement instead.

Reply:
Updated the input in: https://github.com/bitrise-steplib/steps-cocoapods-install/pull/82/commits/e411ea35a4722395ff9516660e4cff092eaafbc8